_lab_members/masters/lucas_march.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: lab_member
+category: Masters
+title: Lucas March
+degree: MA in Music Technology
+photo: assets/lab_members/lucasphoto.jpg
```

Comment:
Looking at others' templates, I think this should be just `lucasphoto.jpg`, without `assets/...`